### PR TITLE
Add ico support

### PR DESCRIPTION
**Pull request by hlohaus**
### 1. Why is this change necessary?

Try setting an icon in the theme settings.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Core/Content/Media/Thumbnail/ThumbnailService.php [hunk 1]
```diff
@@ -362,7 +362,8 @@ private function thumbnailsAreGeneratable(MediaEntity $media): bool
     {
         return $media->getMediaType() instanceof ImageType
             && !$media->getMediaType()->is(ImageType::VECTOR_GRAPHIC)
-            && !$media->getMediaType()->is(ImageType::ANIMATED);
+            && !$media->getMediaType()->is(ImageType::ANIMATED)
+            && $media->getMimeType() !== 'image/x-icon';
```
Comment (mitelg) [inline src/Core/Content/Media/Thumbnail/ThumbnailService.php]: is there no constant for this?

Comment (hlohaus) [reply src/Core/Content/Media/Thumbnail/ThumbnailService.php]: From who? PHP or Vendor Package? :)

Comment (mitelg) [reply src/Core/Content/Media/Thumbnail/ThumbnailService.php]: just like `ImageType::ANIMATED`

**Comment by lernhart** [comment]
Thank you for your contribution @hlohaus :+1: :tada:

**Pull request closed without merging**